diff -aurN MCP-plugins-0.0.2.orig/Makefile MCP-plugins-0.0.2/Makefile
--- MCP-plugins-0.0.2.orig/Makefile	2003-05-19 23:39:33.000000000 +0200
+++ MCP-plugins-0.0.2/Makefile	2003-06-19 18:52:51.000000000 +0200
@@ -28,7 +28,7 @@
 
 
 install:
-	cp  *.so /usr/lib/ladspa
+	cp  *.so $(DESTDIR)/usr/lib/ladspa
 
 clean:
 	rm -f *~ *.o *.so
diff -aurN MCP-plugins-0.0.2.orig/ladspaplugin.h MCP-plugins-0.0.2/ladspaplugin.h
--- MCP-plugins-0.0.2.orig/ladspaplugin.h	2003-04-24 19:03:19.000000000 +0200
+++ MCP-plugins-0.0.2/ladspaplugin.h	2003-06-19 18:54:42.000000000 +0200
@@ -21,7 +21,7 @@
 #define __LADSPAPLUGIN_H
 
 
-#include "ladspa.h" // use local version
+#include <ladspa.h>
 
 
 
