diff -aurN MCP-plugins.orig/ladspaplugin.h MCP-plugins/ladspaplugin.h
--- MCP-plugins.orig/ladspaplugin.h	2003-04-24 19:03:19.000000000 +0200
+++ MCP-plugins/ladspaplugin.h	2003-10-26 01:43:23.326190032 +0200
@@ -21,7 +21,7 @@
 #define __LADSPAPLUGIN_H
 
 
-#include "ladspa.h" // use local version
+#include <ladspa.h>
 
 
 
diff -aurN MCP-plugins.orig/Makefile MCP-plugins/Makefile
--- MCP-plugins.orig/Makefile	2003-07-29 01:16:09.000000000 +0200
+++ MCP-plugins/Makefile	2003-10-26 01:43:02.379374432 +0200
@@ -28,7 +28,7 @@
 
 
 install:
-	cp  *.so /usr/lib/ladspa
+	cp  *.so $(DESTDIR)/usr/lib/ladspa
 
 clean:
 	rm -f *~ *.o *.so
