--- MCP-plugins-0.4.0/Makefile.orig	2022-01-22 22:35:04.828194557 +0100
+++ MCP-plugins-0.4.0/Makefile	2022-01-22 22:41:24.752802995 +0100
@@ -15,42 +15,44 @@
 #  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
 
 
-CPPFLAGS += -I. -fPIC -D_REENTRANT -Wall -O3
+CXX = g++
+CPPFLAGS += -I. -fPIC -D_REENTRANT -Wall
 
+INSTALL_PLUGINS_DIR=/usr/lib/ladspa
 
 all:	mvclpf24.so mvchpf24.so cs_phaser.so cs_chorus.so
 
 
 mvclpf24.so:	mvclpf24.o mvclpf24_if.o exp2ap.o
-	g++ -shared  mvclpf24.o mvclpf24_if.o exp2ap.o -o mvclpf24.so
+	$(CXX) $(LDFLAGS) $(CXXFLAGS) -shared  mvclpf24.o mvclpf24_if.o exp2ap.o -o mvclpf24.so
 
 mvclpf24.o:	ladspaplugin.h mvclpf24.h
 mvclpf24_if.o:	ladspaplugin.h mvclpf24.h
 
 
 mvchpf24.so:	mvchpf24.o mvchpf24_if.o exp2ap.o
-	g++ -shared  mvchpf24.o mvchpf24_if.o exp2ap.o -o mvchpf24.so
+	$(CXX) $(LDFLAGS) $(CXXFLAGS) -shared  mvchpf24.o mvchpf24_if.o exp2ap.o -o mvchpf24.so
 
 mvchpf24.o:	ladspaplugin.h mvchpf24.h
 mvchpf24_if.o:	ladspaplugin.h mvchpf24.h
 
 
 cs_phaser.so:	cs_phaser.o cs_phaser_if.o exp2ap.o
-	g++ -shared cs_phaser.o cs_phaser_if.o exp2ap.o -o cs_phaser.so
+	$(CXX) $(LDFLAGS) $(CXXFLAGS) -shared cs_phaser.o cs_phaser_if.o exp2ap.o -o cs_phaser.so
 
 cs_phaser.o:	ladspaplugin.h cs_phaser.h
 cs_phaser_if.o:	ladspaplugin.h cs_phaser.h
 
 
 cs_chorus.so:	cs_chorus.o cs_chorus_if.o exp2ap.o
-	g++ -shared  cs_chorus.o cs_chorus_if.o exp2ap.o -o cs_chorus.so
+	$(CXX) $(LDFLAGS) $(CXXFLAGS) -shared  cs_chorus.o cs_chorus_if.o exp2ap.o -o cs_chorus.so
 
 cs_chorus.o:	ladspaplugin.h cs_chorus.h
 cs_chorus_if.o:	ladspaplugin.h cs_chorus.h
 
 
 install:	all
-	cp  *.so /usr/lib/ladspa
+	cp -p  *.so $(DESTDIR)$(INSTALL_PLUGINS_DIR)
 
 
 DIR := $(shell basename `pwd`)
diff -ur MCP-plugins-0.3.0.orig/ladspaplugin.h MCP-plugins-0.3.0/ladspaplugin.h
--- MCP-plugins-0.3.0.orig/ladspaplugin.h	2003-12-01 21:18:18.000000000 +0100
+++ MCP-plugins-0.3.0/ladspaplugin.h	2004-10-05 01:55:30.852437288 +0200
@@ -21,7 +21,7 @@
 #define __LADSPAPLUGIN_H
 
 
-#include "ladspa.h" // use local version
+#include <ladspa.h>
 
 
 
